sljit_src/sljitNativeMIPS_common.c

```diff
@@ -2727,9 +2800,9 @@ static SLJIT_INLINE sljit_s32 sljit_emit_fop1_conv_sw_from_f64(struct sljit_comp
 
 	if (FAST_IS_REG(dst)) {
 		FAIL_IF(push_inst(compiler, MFC1 | flags | T(dst) | FS(TMP_FREG1), MOVABLE_INS));
-#if (!defined SLJIT_MIPS_REV || SLJIT_MIPS_REV <= 1)
```

Comment:
What is the problem with this check?

Reply:
There were several versions of it so I standarized in one that is more likely to be correct.

The need for `nop` is only required for MIPS III or older, but SLJIT_MIPS_REV == 1 in MIPS64 corresponds to MIPS IV.  It might be correct for MIPS32 though, but I think it is also unlikely anyone would have a mips32r1 CPU with FPU to test, and indeed until this code change no mips32r1 CPU would have FPU enabled because they have 32bit registers.

Reply:
Shall we make mipsr1 as minimum?

Reply:
brought the check back with a more conservative setting so at least `NOP` could be skipped for most MIPS64 CPUs.